File: src/01-easy/1d-bush-fire/solution.ts
const n = parseInt(readline());
for (let i = 0; i < n; i++) {
    const s = readline(); let d = 0, j = 0;
    while (j < s.length) { if (s[j] === 'f') { d++; j += 3; } else j++; }
    console.log(d);
}

File: src/01-easy/1d-bush-fire/solution.fs
open System
[<EntryPoint>]
let main _ =
    let n = Console.ReadLine() |> int
    for _ in 1..n do
        let s = Console.ReadLine().Trim()
        let mutable d = 0
        let mutable j = 0
        while j < s.Length do
            if s.[j] = 'f' then d <- d+1; j <- j+3 else j <- j+1
        printfn "%d" d
    0
